### PR TITLE
Allow sending to case owner when location is case owner

### PR DESCRIPTION
**Pull request by gcapalbo**
Fixes http://manage.dimagi.com/default.asp?182826

@biyeun

### CODE REVIEW

File: corehq/apps/reminders/models.py [hunk 1]
```diff
@@ -1484,7 +1488,13 @@ def _recipient_lookup(self):
         elif handler.recipient == RECIPIENT_SURVEY_SAMPLE:
             return CommCareCaseGroup.get(handler.sample_id)
         elif handler.recipient == RECIPIENT_OWNER:
-            return get_wrapped_owner(get_owner_id(self.case))
+            owner_id = get_owner_id(self.case)
+
+            location = SQLLocation.by_location_id(owner_id)
+            if location:
+                return [location]
```
Comment (czue) [inline corehq/apps/reminders/models.py]: not too familiar with this part of the code, but any reason you're wrapping it in a list (as opposed to just checking explicitly for the type inside `recipient_is_location`)? assume it's a list elsewhere so you would need to support both structures, but just seems a little odd. might be a little clearer if you pushed the value/list abstraction into `recipient_is_location` instead

Comment (gcapalbo) [inline corehq/apps/reminders/models.py]: @czue it's to match the output of https://github.com/dimagi/commcare-hq/blob/533cb0639e401676bcc5f434fa52de46844cc1d9/corehq/apps/reminders/models.py#L1247

Comment (czue) [inline corehq/apps/reminders/models.py]: gotcha. maybe just rename the function to `recipient_is_list_of_locations` for clarity? or do you think that's silly

Comment (gcapalbo) [inline corehq/apps/reminders/models.py]: fair enough, updated

**Pull request merged**